app/scripts/services/logLinks.js

```diff
@@ -46,7 +46,7 @@ angular.module('openshiftConsole')
 
       // broken up for readability:
       var template = _.template([
-        "/#/discover?",
+        "<%= url %>/#/discover?",
```

Comment:
Do we know if url can ever end in a `/` causing a double forward slash? We should give it a name like `baseURL` if it's not the full URL.

I'm a little confused why we need to do this now... How did the base URL get prepended before?

Reply:
OK, well if either is a valid config value, we will want to make sure there is no double slash in the URL. Do you'd want some JS to handle that.